src_assets/linux/misc/60-sunshine.rules

```diff
@@ -1,2 +1,14 @@
+# Allows Sunshine to acces /dev/uinput
 KERNEL=="uinput", SUBSYSTEM=="misc", OPTIONS+="static_node=uinput", TAG+="uaccess"
+
+# Allows Sunshine to access /dev/uhid
 KERNEL=="uhid", TAG+="uaccess"
+
+# Move virtual keyboard and mouse into a different seat
+SUBSYSTEMS=="input", ATTRS{id/vendor}=="ab00", ENV{ID_SEAT}="seat9", MODE="0660", TAG+="uaccess"
+
+# Joypads
+SUBSYSTEMS=="input", ATTRS{name}=="Sunshine X-Box One (virtual) pad", MODE="0660", TAG+="uaccess"
+SUBSYSTEMS=="input", ATTRS{name}=="Sunshine PS5 (virtual) pad", MODE="0660", TAG+="uaccess"
+SUBSYSTEMS=="input", ATTRS{name}=="Sunshine gamepad (virtual) motion sensors", MODE="0660", TAG+="uaccess"
+SUBSYSTEMS=="input", ATTRS{name}=="Sunshine Nintendo (virtual) pad", MODE="0660", TAG+="uaccess"
```

Comment:
For the DualSense it would be better to have
```
KERNEL=="hidraw*" ATTRS{name}=="Sunshine PS5 (virtual) pad" MODE="0660", TAG+="uaccess"
```
Following the [official Steam rules](https://github.com/ValveSoftware/steam-devices/blob/e2971e45063f6b327ccedbf18e168bda6749155c/60-steam-input.rules#L38)